modules/installation-configuration-parameters.adoc

```diff
@@ -627,6 +627,12 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifdef::aws[]
+|`platform.aws.lbType`
+|Required to set the NLB load balancer type. The default value is Classic LB. If you use NLB load balancing, the ingress operator creates the load balancer for the default ingress controller.
```

Comment:
It can either be "NLB" or "Classic" cc @Miciah

Reply:
@miheer And will the `oc edit ingress.config/cluster -o yaml` command have to be run to save or update the cluster configuration file that the installer outputs?